app/models/benefit_model.rb

```diff
@@ -5,6 +5,14 @@ class BenefitModel
 
   FREQUENCY_OPTIONS = %w[every_week every_two_weeks every_four_weeks monthly].freeze
 
+  PASSPORTED_BENEFITS = [
+    "Universal Credit",
+    "Income Support",
+    "Jobseeker's Allowance (JSA)",
```

Comment:
I think CFE return's `Jobseeker’s Allowance (JSA)` (different type of apostrophe). I wonder if, to avoid this sort of fragility, it would be better to exclude benefits based on the _label_ attribute (`jobseekers_allowance` in this case), which has less fiddly formatting.